### ISSUE TITLE
bug in impact premium

### ISSUE THREAD

**Issue by IHackedAmazon** [cabaletta/baritone#1943]
when you're using impact premium and walks on chests, it crashes the game 
vid : https://www.youtube.com/watch?v=-DWpA0WI6do
log : https://github.com/ImpactDevelopment/ImpactIssues/files/5047552/2020-08-09-7.log
(dominika said its baritone issue so im reporting it here)

**Comment by TheK0tYaRa** [comment]
Well isn't lazy impact devs that forget to update their client when baritone is fixed for half a year

**Comment by dominikaaaa** [comment]
what????

**Comment by TheK0tYaRa** [comment]
Remember that bug that occurred with buildcraft pipes or IC2 wires before? It worked the same way sometimes with chests if they are in chess pattern.

**Comment by dominikaaaa** [comment]
No, I don't remember every single obscure bug that happens with random modpacks
